napari/_qt/qt_main_window.py

```diff
@@ -124,6 +124,12 @@ def _toggle_menubar_visible(self):
             self.main_menu.setVisible(True)
             self._main_menu_shortcut.setEnabled(False)
 
+    def _toggle_theme(self):
```

Comment:
I'd prefer to see this function look like the toggle_theme function in our `napari/keybindings.py` https://github.com/napari/napari/blob/c01024f151546f7a2ada9727572efb3e65fa4f0f/napari/keybindings.py#L15
Can you replace the code here with that code (keeping the name `_toggle_theme` and remove the code around `light` and `dark`. The reason for this is that if people add more themes it will allow them to cycle through them instead of just having to alternate between our two fixed ones

Reply:
Ah, also this function should maybe live inside the `ViewerModel` https://github.com/napari/napari/blob/master/napari/components/viewer_model.py instead of `qt_main_window` as it only needs to reference the Viewer

Reply:
I like it. I copied this and moved it to `viewer_model`